src/main/java/net/neoforged/neoforge/event/village/VillagerTradesEvent.java

```diff
@@ -28,10 +29,12 @@
 public class VillagerTradesEvent extends Event {
     protected Int2ObjectMap<List<ItemListing>> trades;
     protected VillagerProfession type;
+    protected RegistryAccess registryAccess;
```

Comment:
Any reason it's `protected` here (and below) instead of `private`? Protected fields are API-relevant as mods could subclass.

Reply:
It was more I didn't know if I should use `protected` or `private` here, so I just matched, will turn both to private now

Reply:
I would do so. Not so much to protect them from accidental change or help the JVM optimise the code, but for semantic reasons to make it easy to see that they are not changeable values (results) of the event.

Reply:
That's all done then, I assumed as much and I did that update a couple hours XD